<launch>
  <node name="bag_player" pkg="rosbag" type="play" args="-r 0.1 $(find viso2_stereo_server)/bag/test.bag" required="true"/>

  <node name="service_server" pkg="viso2_stereo_server" type="viso2_stereo_server" output="screen"/>
  <node name="service_client" pkg="viso2_stereo_server" type="viso2_service_test_client" output="screen">
    <remap from="left_image" to="/stereo_robot/mobile_base/camera/left/image_raw"/>
    <remap from="right_image" to="/stereo_robot/mobile_base/camera/right/image_raw"/>
    <remap from="estimate_motion_from_stereo" to="/service_server/estimate_motion_from_stereo"/>

    <param name="image_transport" value="compressed"/>
  </node>
</launch>

